misc/language-tour-guide/src/content/1-basic-types-and-operations/structural-comparison.mdx

```diff
@@ -0,0 +1,17 @@
+export const frontmatter = { order: 20 };
+
+# Structural comparison
+
+Elixir also provides `==`, `!=`, `<=`, `>=`, `<` and `>` as comparison operators. We can compare numbers, atoms, strings, booleans, etc. Integers and floats compare the same if they have the same value, but you can use the strict comparison operator `===` and `!==` if you want to distinguish between integers and floats.
```

Comment:
Important thing here is the order of terms when comparing and caveats that come from that